modules/lang-painless/src/main/java/org/elasticsearch/painless/node/ELambda.java

```diff
@@ -100,7 +101,7 @@ void extractVariables(Set<String> variables) {
 
     @Override
     void analyze(Locals locals) {
-        final Definition.Type returnType;
+        final Type returnType;
         final List<String> actualParamTypeStrs;
```

Comment:
do you think this is the right tradeoff in this file? It causes the much longer `org.objectweb.asm.Type` many times in the file, including making lines so long they must be unnaturally split. Maybe we should rename Definition.Type to something that does not conflict? I would also rename Compiler to something that does not conflict with java.lang.Compiler! We can do this separately, but these conflicts are a hassle...